### DISCUSSION TITLE
Is there any possibility of integrating Claude through Poe?

### DISCUSSION THREAD

**Discussion by artificialguybr** [acheong08/EdgeGPT#219] [General]
There are already some reverse API repos for Poe (poe.com) that have the Claude model and some ChatGPT models for free.
/
The Claude model has the advantage of greater creativity and better ability to create creative texts. It's interesting and has no limit via POE.

There are already some repos that provide this reverse service, such as:
-https://github.com/vaibhavk97/Poe
-https://github.com/ading2210/poe-api

Thank you for all the support provided! Have an excellent day!

**Comment by acheong08** [comment]
> There are already some repos that provide this reverse service, such as:

It is out of scope for this specific repo and I would rather not intrude on their work. Use their libraries instead
